Project/VetAppointmentAPI/VetAppointmentAPI/VetAppointment.Application/Repositories/Interfaces/IDrugRepository.cs

```diff
@@ -5,5 +5,7 @@ namespace VetAppointment.Application.Repositories.Interfaces
 {
     public interface IDrugRepository : IBaseRepository<Drug>
     {
+        public Task<Drug> UpdateAsync(Drug drug);
```

Comment:
These two methods also go in the IBaseRepository interface (all will be templetized, similarly to what already is in the Base classes).